### PR TITLE
[debian] fix nlohmann-json-dev package name

### PR DESCRIPTION
**Pull request by Rechi**
@wsnipex FYI

### CODE REVIEW

**Comment by Rechi** [comment]
You still have a wrong understanding how Ubuntu (Debian) packaging works. You can't choose which version you want, either it's available or not.

This PR just adds version constraints and a possible alternative package name, as Debian folks decided to give the 3.* version a new package name (what I've overlooked).

**Comment by wsnipex** [comment]
this will only work for ubuntu 18.10, as the package doesn't exist in <=18.04.
I'll have to backport it first.

**Comment by phunkyfish** [comment]
Thanks @Rechi for explaining.

File: debian/control [hunk 1]
```diff
@@ -3,7 +3,7 @@ Priority: extra
 Maintainer: Cristiano A. Silva <hudokkow@gmail.com>
 Build-Depends: debhelper (>= 9.0.0), cmake, libtinyxml-dev,
                libkodiplatform-dev (>= 16.0.0), kodi-addon-dev,
-               nlohmann-json-dev
+               nlohmann-json-dev (>= 3.1) | nlohmann-json3-dev (>= 3.1)
```
[the remark this answers is not in the document]
Comment (Rechi) [reply debian/control]: I tested with 3.1.0 and it compiled.

**Pull request merged**